sdk-api.yaml

```diff
@@ -567,6 +619,13 @@ components:
           type: string
           nullable: false
           example: "admin"
+    Users:
+      type: object
+      properties:
+        items:
+          type: array
+          items:
+            $ref: '#/components/schemas/User'
```

Comment:
👍 on using `items`:

```
{
  items: [{...}, {...}]
}
```

 We can standardized on something else (although it may be too late), but it seems good enough for now.

Reply:
`items` works for me. I think I've seen this around other APIs and it seems sensible.